droughtpipeline/load.py

```diff
@@ -417,11 +406,71 @@ def send_to_ibf_api(
                             self.ibf_api_post_request("admin-area-dynamic-data/exposure", body=body )
                     processed_pcodes = list(set(processed_pcodes))
 
+                elif lead_time_event == 0:
```

Comment:
I think this setup does not allow for the fallback if no 1-month data is found? I would set it up:
- within the 'IF leadTime in range (0,4)'
- start with an additional If leadtime = 0
- get the pre-existing 1-month data
- if not available OR if leadtime > 0: then get data in regular way
- then in call cases, process data the same way. That allows also for less code duplication I think. That should be possible if I'm correct?

Let's call about this idea if unclear.